readthedocs-theme/templates/article.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% block html_lang %}{{ article.lang }}{% endblock %}
+
+
+{% block title %}{{ article.title|striptags  }} - {{ SITENAME }}{% endblock %}
+
+
+{% block content %}
+
+
+  <!-- Article -->
+  <section class="ui vertical segment padded" id="article">
+    
+    <div class="ui container">
+      <div class="ui grid centered">
+        <div class="column twelve wide computer fourteen wide tablet sixteen wide mobile">
+
+
+          <h1 class="ui header">
+            <span class="ui header big">{{ article.title }}</span>
```

Comment:
`big` is really big here, it dominates the top of the page a bit:

![image](https://user-images.githubusercontent.com/1140183/152918182-48756190-3487-4953-8763-f76fac61454d.png)

This looks a bit better, at least on the test article heading.

```suggestion
            <span class="ui header">{{ article.title }}</span>
```